common/src/certificate.rs

```diff
@@ -1,6 +1,7 @@
 use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
 
-#[derive(Deserialize, Serialize, Debug)]
+#[derive(Deserialize, Serialize, Debug, ToSchema)]
```

Comment:
To keep `common` deps light, we want to feature gate "utoipa" and use `#[cfg_attr("utoipa", derive(utoipa::ToSchema))]` (inlined to not need `use` with cfg flag ;) )

Reply:
Put that into 28e195d46791